analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0-SPDX2TV.spdx

```diff
@@ -0,0 +1,1455 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_webrtcHacks-adapter-v7.3.0.tar.gz_1717150416.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Sascha Dierberg
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of webrtcHacks-adapter 7.3.0.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-05-31T10:13:37Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: webrtcHacks-adapter-v7.3.0.tar.gz
+PackageFileName: webrtcHacks-adapter-v7.3.0.tar.gz
+SPDXID: SPDXRef-upload120
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: 625a6b6f19728d41e2bd214fad366de02fb2d68c
+PackageChecksum: SHA1: 7d861806e2669c0a67e16738070d05828bb849eb
+PackageChecksum: SHA256: 212c3e62fcef8cb976c62848b193c82f2c74f0cfa398a6838dfc8ca4fa96ee5c
+PackageChecksum: MD5: ac11fb6ae2292145e88eda4908f7ade0
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-BSD-3-Clause
```

Comment:
Please correct the PacvkageLicenseDeclared, it must be:
LicenseRef-BSD-3-Clause-5d180954454293e3705aecb03706c1e6
Instead of LicenseRef-BSD-3-Clause